omod/src/main/webapp/resources/htmlforms/HMIS-005-IntegratedAntenatalRegister.xml

```diff
@@ -1,4 +1,51 @@
 <htmlform formName="HMIS 005 Integrated Antenatal Register" formDescription="Entry form for patients enrolled for ANC services" formEncounterType="044daI6d-f80e-48fe-aba9-037f241905Pe" formUuid="362e296b-1315-48a9-8de6-5f4cc3cad314" formVersion="3.0.0" formAddMetadata="yes" formUILocation="patientDashboard.visitActions" formOrder="8" formIcon="icon-medkit" formShowIf="(visit.active || !visit.active) &amp;&amp; patient.person.gender=='F' &amp;&amp; patient.person.age &gt; 9 &amp;&amp; patient.person.dead==false &amp;&amp; sessionLocation.uuid=='e9bc61b5-69ff-414b-9cf0-0c22d6dfca2b'" formDisplayStyle="Standard" formLabel="ANC Encounter Form">
+    <script type="text/javascript">
+    //Disable encounter date field.
+            var disableEncounterDate = function() {
+                $encounterDateField = jq('#encounterDate').find('input[type="text"]');
+                $encounterDateField.addClass('disabled');
+                $encounterDateField.attr('disabled', 'disabled');
+            }
+
+     if (jQuery) {
+        jq(document).ready(function () {
+            //Disable encounter date field
+            disableEncounterDate();
+
+            // Handle viral load
+            
+            var disableViralLoadFields = function(){
+                jq('#viral-load').find('input[type="text"]').attr("disabled", "true");
+                jq('#viral-load').find('input[type="text"]').addClass('disabled');
+
+                jq('#viral-load-results').find('select').attr("disabled", "true");
+                jq('#viral-load-results').find('select').addClass('disabled');
+                
+                jq('#viral-load-status').find('select').attr("disabled", "true");
+                jq('#viral-load-status').find('select').addClass('disabled');
+            }
+
+            var enableViralLoadFields = function(){
+                jq('#viral-load').find('input[type="text"]').removeAttr("disabled");
+                jq('#viral-load').find('input[type="text"]').removeClass('disabled');
+
+                jq('#viral-load-results').find('select').removeAttr("disabled");
+                jq('#viral-load-results').find('select').removeClass('disabled');
+                
+                jq('#viral-load-status').find('select').removeAttr("disabled");
+                jq('#viral-load-status').find('select').removeClass('disabled');
+            }
+            disableViralLoadFields();
+            jq('#163023').find('#w44').change(function($event){
```

Comment:
@SokoPaulSokool you can attach a JS file by using the functionality at https://wiki.openmrs.org/display/docs/HTML+Form+Entry+JavaScript+Reference#HTMLFormEntryJavaScriptReference-getField(idAndProperty) 

@solemabrothers